tests/BenchmarkDotNet.IntegrationTests/ExporterIOTests.cs

```diff
@@ -90,5 +91,47 @@ public override void ExportToLog(Summary summary, ILogger logger)
                 ExportCount++;
             }
         }
+
+        [Fact]
+        public void ExporterUsesFullyQualifiedTypeNameAsFileName()
+        {
+            var exporter = new MockExporter();
+            var summary = CanExecute<ClassA>();
```

Comment:
could you create some `Summary` without running the benchmarks?

We are currently having a problem with too long builds. We have too many integration tests that run benchmarks and we exceed the 1h we have in our current plan :/

Reply:
Yes I'll look into it! AppVeyor currently reports these durations for the added tests:

Test | Duration
----- | ---------
ExporterUsesFullyQualifiedTypeNameAsFileName | 28 sec 293 ms
ExporterUsesSummaryTitleAsFileNameWhenBenchmarksJoinedToSingleSummary | 3 min 51 sec 219 ms

I suppose its possible to get these down to a couple of milliseconds each, if I mock out the `Summary` correctly.

Reply:
> I suppose its possible to get these down to a couple of milliseconds each, if I mock out the Summary correctly.

exactly! thanks for understanding!